src/lib/storage/vector_compression/bitpacking/bitpacking_vector.cpp

```diff
@@ -26,7 +26,7 @@ std::unique_ptr<const BaseCompressedVector> BitPackingVector::on_copy_using_allo
     const PolymorphicAllocator<size_t>& alloc) const {
   auto data_copy = pmr_compact_vector(_data.bits(), _data.size(), alloc);
   // zero initialize the compactvectors memory, see bitpacking_compressor.cpp
-  // std::fill_n(data_copy.get(), data_copy.bytes() / 8, 0);
+  std::fill_n(data_copy.get(), data_copy.bytes() / sizeof(*data.get()), 0);
```

Comment:
You still somehow assume that 0 is of type `uint64_t`, don't you?
I don't like this usage of `std::fill_n`, if you want to keep it this way, please add sufficient comments.

Reply:
changed it to `0ULL`, sorry.

Reply:
This is the same discussion as in the` bitpacking_compressor.cpp` https://github.com/hyrise/hyrise/pull/2350#discussion_r626476001. I improved the docs there further to reflect that compact::vector unfortunately does not provide a better interface and therefore we need to work this raw pointer to it's internal memory. Is it okay if I just refer to the documentation there?

Reply:
I am fully aware that this usage of `std::fill_n` is quite ugly and we're talking about it since a week now :P. If you can point me to a better solution I'd be happy to change it, I just did not find anything better.

Reply:
closing this as you said "PR von meiner Seite aus durch" and did not request other changes